topics/climate-change-mitigation/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: reducing-ghg-emissions, reducing-carbon-emissions, recapturing-ghg-emissions, recapturing-carbon-emissions
+created_by: Humanity
+display_name: Climate change mitigation
+github_url: https://github.com/topics/climate-change-mitigation
```

Comment:
It's a little odd to have the `github_url` set to the topic page itself! Usually this is a URL for some repo that's central to the topic, like the React repo for the React topic. You can just omit it for topics without a clear repo like this one. I see that we missed it for the climate-change topic, though, so we can merge this as-is and maybe clear both of them in a follow-up PR?

Reply:
Sure, will do!